### PR TITLE
Remove uitk-theme custom element

### PR DESCRIPTION
**Pull request by ImmanuelBaskaran**
Remove uitk-theme custom element and replace it with div element. The custom element may cause confusion during ADA checks and server side rendering.

### CODE REVIEW

**Review by joshwooding — APPROVED** [review]

File: packages/core/src/__tests__/__e2e__/toolkit-provider/ToolkitProvider.cy.tsx [hunk 1]
```diff
@@ -43,16 +41,16 @@ const TestComponent = ({
...
       cy.mount(
         <ToolkitProvider>
           <TestComponent />
         </ToolkitProvider>
       );
 
-      cy.get("uitk-theme")
+      cy.get("div.uitk-theme")
         .should("have.length", 2)
-        .and("have.attr", "class", "uitk-light uitk-density-medium");
```
Comment (joshwooding) [inline packages/core/src/__tests__/__e2e__/toolkit-provider/ToolkitProvider.cy.tsx:55]: I think we should still check for the light theme here?

**Review by joshwooding — CHANGES_REQUESTED** [review]
Looks good, just left one comment

File: packages/core/src/toolkit-provider/ToolkitProvider.tsx [hunk 2]
```diff
@@ -25,31 +23,13 @@ export interface ToolkitContextProps {
...
 
 export const BreakpointContext =
   createContext<Breakpoints>(DEFAULT_BREAKPOINTS);
 
-export const ToolkitContext = createContext<ToolkitContextProps>({
-  density: undefined,
-  themes: [],
-  breakpoints: DEFAULT_BREAKPOINTS,
-});
-
```
Comment (ZarkoPernar) [inline packages/core/src/toolkit-provider/ToolkitProvider.tsx:52]: Why are we removing this?

Comment (ImmanuelBaskaran) [reply packages/core/src/toolkit-provider/ToolkitProvider.tsx:52]: That was an accidental leftover from splitting the ToolkitContext into separate contexts for Density, Breakpoints and theme ( see PR #602  )

**Review by ZarkoPernar — APPROVED** [review]

**Review by joshwooding — APPROVED** [review]

**Pull request merged**